Kernel/FileSystem/SysFS/Subsystems/Kernel/Variables/FixedStringBufferVariable.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2023, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/FileSystem/SysFS/Subsystems/Kernel/Variables/FixedStringBufferVariable.h>
+#include <Kernel/Sections.h>
+#include <Kernel/Tasks/Process.h>
+
+namespace Kernel {
+
+ErrorOr<void> SysFSSystemFixedStringBufferVariable::try_generate(KBufferBuilder& builder)
+{
+    auto string_value = TRY(value());
+    return builder.appendff("{}\n", string_value->view());
+}
+
+ErrorOr<size_t> SysFSSystemFixedStringBufferVariable::write_bytes(off_t offset, size_t count, UserOrKernelBuffer const& buffer, OpenFileDescription*)
+{
+    if (offset != 0)
+        return EINVAL;
+    if (m_storage_during_write_buffer_size < count)
+        return E2BIG;
+    // NOTE: If we are in a jail, don't let the current process to change the variable.
```

Comment:
`don't let the current process to change the variable` -> `don't let the current process change the variable`